app/assets/javascripts/modules/collapsible_collection.js

```diff
@@ -18,8 +18,8 @@
 
     if(this.$sections.length > 0) {
       this.$sections.each(this.initCollapsible.bind(this));
-      this.$openAll = $("<a href='#' aria-hidden=true>Open all</a>");
-      this.$closeAll = $("<a href='#' aria-hidden=true>Close all</a>");
+      this.$openAll = $("<button type='button'>Open all</button>");
+      this.$closeAll = $("<button type='button'>Close all</button>");
```

Comment:
🎉 Thanks for this!

(First time I've seen `type="button"` - think I usually just do `<button>`, and provided it's in a form, it doesn't need `preventDefault`. I guess adding `type="button"` means you've been able to remove the `preventDefault` here: https://github.com/alphagov/manuals-frontend/pull/642/files#diff-8f262d6c492018c739c12c25a4a34efeL27 ?)

Reply:
you're correct that type=button isn't strictly necessary as it's unlikely this block will ever be in a form. Will remove as i've copied it over from https://govuk-publishing-components.herokuapp.com/component-guide/accordion

Reply:
type=button removed